test/system/src/test/java/io/pravega/test/system/ConsumptionBasedRetentionWithMultipleReaderGroupsTest.java

```diff
@@ -280,7 +242,185 @@ public void multipleSubscriberCBRTest() throws Exception {
         // Check to make sure truncation happened at max stream cut
         AssertExtensions.assertEventuallyEquals("Truncation did not take place at offset 330.", true, () -> controller.getSegmentsAtTime(
                         new StreamImpl(SCOPE, STREAM), 0L).join().values().stream().anyMatch(off -> off == 330),
-                5000, 3 * 60 * 1000L);
+                5000, 2 * 60 * 1000L);
+
+    }
+
+    @Test
+    public void updateRetentionPolicyForCBRTest() throws Exception {
+        final ClientConfig clientConfig = Utils.buildClientConfig(controllerURI);
```

Comment:
Can we make clientConfig as an instance variable as it is common in both test cases?

Reply:
clientConfig requires controllerURI which we get once the controller service starts. Thus made it as local variable in both the test cases.